pkgs/development/libraries/spice-gtk/default.nix

```diff
@@ -16,7 +17,7 @@
 , libdrm
 , libjpeg_turbo
 , libopus
-, libsoup
+, libsoup_3
```

Comment:
The libsoup 3 migration unfortunately is not that simple. It needs to be switched in all the reverse dependencies at the same time, see https://discourse.gnome.org/t/phodav-transition-notes/10483 and #182618

Reply:
I see. I'll push another commit that reverts to libsoup.

Reply:
Reverted to libsoup now.